packages/database/src/TupaiaDatabase.js

```diff
@@ -45,7 +45,7 @@ export const JOIN_TYPES = {
 };
 
 // list valid behaviour so we can validate against sql injection
-const VALID_CAST_TYPES = ['text', 'text[]', 'date'];
+const VALID_CAST_TYPES = ['text', 'text[]', 'date', 'entity_type[]'];
```

Comment:
I have mixed feelings about this...adding a piece of very Tupaia specific metadata into the otherwise generic `TupaiaDatabase`. The name of the class would suggest it's ok, but it is the first example we have of breaking the genericism